### DISCUSSION TITLE
Please add all possible config statements to config template

### DISCUSSION THREAD

**Discussion by elproko** [thirtythreeforty/neolink#121] [Ideas]
First of all: THANKS for your titanic work for reverse engeenering this proto for us, and then making a workaround !!!

An idea:
For preventing any issues please consider to add all possible config statements (hashed/commented out) to template config file.
I faced issues because of missing stream = "something" with D400 cam.

I believe https://github.com/thirtythreeforty/neolink/issues/86 - has the same source.

**Comment by QuantumEntangledAndy** [comment]
By default `stream="both"` for most users this is the best default. I don't think we should uncommented this and make it `mainStream` for all, since this is not a reasonable default for most.

**Reply by elproko** [reply]
I didn't write that all settings should be present in example config as **active**.
Instead I wrote they should be there as commented - not active.
It is just for let users know that some of them are important.

**Comment by twistedddx** [comment]
I have a PR that might help to explain more settings in the example config:
https://github.com/thirtythreeforty/neolink/pull/122
